gcsmigrate/k8s/base/final_migrate_job.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: lemursync-final
+spec:
+  schedule: "5,15,25,35,45,55 * * * *"
```

Comment:
just fyi you can also write this like
5-59/10 * * * *

Reply:
I wanted this to run after the migrate job, every 10 minutes on the 5. Will that work?